components/WrapperForm.tsx

```diff
@@ -206,26 +174,16 @@ const WrapperForm: React.FC<WrapperFormProps> = ({ action }) => {
         </Button>
       </form>
 
-      <Flex
-        color='white'
-        justifyContent='center'
-        mt={
-          isSuccessDeposit ||
-          isSuccessWithdraw ||
-          // isErrorDeposit ||
-          // isErrorWithdraw ||
-          isLoadingDeposit ||
-          isLoadingWithdraw
-            ? '5'
-            : 0
-        }
-      >
-        {isLoadingDeposit || isLoadingWithdraw ? (
-          <Toast title='Pending transaction' description={pendingMsg} />
+      <Flex color='white' justifyContent='center'>
+        {statusDeposit === 'loading' || statusWithdraw === 'loading' ? (
+          <Toast
+            title='Pending transaction'
+            description='Wait a moment while your transaction is being mined...'
+          />
         ) : null}
-        {isLoadingDeposit || isLoadingWithdraw ? null : isSuccessDeposit ||
-          isSuccessWithdraw ? (
-          <Toast title='Success!' description={successMsg} type='success' />
+        {(statusDeposit === 'success' && statusWithdraw !== 'loading') ||
+        (statusWithdraw === 'success' && statusDeposit !== 'loading') ? (
+          <Toast title='Success!' description={successMessage} type='success' />
```

Comment:
this is close, but we want to use the `useToast` hook to invoke this. This is just the renderable component.

Reply:
allows you to get rid of `statusWithdraw/statusDeposit` I think because you can call the toast from your `useWaitTransaction` hook in `useWithdraw/useDeposit`

Reply:
the `Toast` component expects a string for description, so we can handle that type tweak in the design system or update here.